### ISSUE TITLE
Accessibility - Cookie banner should have focus first (SC 2.4.3)

### ISSUE THREAD

**Issue by LaurensBurger** [open-formulieren/open-forms#5455]
### Thema / Theme

Frontend

### Omschrijving / Description

DB: 116

The cookie banner will present itself on top of all other components. Depending on the amount of steps and login options at the beginning of a form it might take around 10/20 "tabs" before the banner is reached with keyboard navigation.
While a form can be filled in without a choice of cookies - the banner itself is blocking the content beneath it - also blocking the focus of the keyboard. 

Example with 16 tabs to reach banner with a lot of focus being hidden behind the banner: 

<img width="1849" height="891" alt="Image" src="https://github.com/user-attachments/assets/cf3b1817-a3ce-4b62-9471-10d3c262bf4d" />
